web/main.js

```diff
@@ -119,7 +119,9 @@ let receive = e => {
 		receiving.data = new Uint8Array(receiving.size);
 		receiving.offset = 0;
 		receiving.li = document.createElement('li');
-		receiving.li.appendChild(document.createTextNode(`↓ ${receiving.name}`));
+		receiving.li.appendChild(document.createElement("a"));
+		receiving.a = receiving.li.getElementsByTagName("a")[0];
+		receiving.a.innerHTML = `↓ ${receiving.name}`;
```

Comment:
This can be vulnerable to XSS, if the name has html tags in it. To escape it:
```
receiving.a.appendChild(document.createTextNode(`↓ ${receiving.name}`));
```